src/advanced/bitcoin-anchoring-without-segwit.md

```diff
@@ -0,0 +1,527 @@
+# Anchoring Service (without SegWit)
+
+<!-- cspell:ignore bitcoind,blockhash,lects,satoshis,txid,utxo,utxos -->
+
+The anchoring service is developed to increase product security and
```

Comment:
I'd suggest to mention in this paragraph that the page describes the anchoring service for Exonum <0.10 in order for the page to be more SEO-friendly. Also, it could make sense to link to the "newer" page.